pkg/neonvm/controllers/vm_controller.go

```diff
@@ -1136,6 +1152,7 @@ func annotationsForVirtualMachine(vm *vmv1.VirtualMachine) map[string]string {
 
 func affinityForVirtualMachine(vm *vmv1.VirtualMachine) *corev1.Affinity {
 	a := vm.Spec.Affinity
+
```

Comment:
nit: unintended diff?

Reply:
probably automatically added it for readability

Reply:
rolled back